src/unity/schemas/Organization.yml

```diff
@@ -1,23 +1,37 @@
 type: object
 properties:
-  idpeId:
-    type: string
-    description: idpe ID of the org
   id:
-    type: number
-    description: quartz ID of the org
+    type: string
+    description: ID of the org
   name:
     type: string
     description: name of the org
-  region:
+  description:
+    type: string
+    description: the description of the organization
+  creationDate:
+    type: string
+    format: date-time
+    description: the date the organization was created
+  regionCode:
+    type: string
+    description: the code associated with the region
+  regionName:
     type: string
-    description: region of the org
+    description: the name of the region
   provider:
     type: string
     description: provider of the org
-  date:
+  isRegionBeta:
+    type: boolean
+    description: is the region of the organization in beta
+  clusterHost:
     type: string
-    description: date org was created
-  account:
-    $ref: './RelatedAccount.yml'
-required: [account, id, idpeId, region, provider, date]
+    description: the url of the current cluster
+  isActive:
```

Comment:
I think we're going to want a separate schema (something like `OrganizationSummary`) for the `GET /orgs` list endpoint that just returns the `id`, `name`, `isActive`, and `isDefault`. Then we can remove `isActive` and `isDefault` from this model. Otherwise the list endpoint will be returning way more information than needed, especially as we start moving/adding more org data to Quartz.

Reply:
What other org data is going to be added to quartz? I don't think it is worth adding a new schema to get rid of two fields. If we know for sure there are a lot of fields that will be added, then I agree with you. However, if we don't know what those fields are / will be, I would prefer to wait to split out the schemas. I can also see a situation where we get the list of orgs for a user and would like to know which ones are active / are the default.

Reply:
`isActive` and `isDefault` take extra work to compute and inject into the structs that we return. I know we debated using a payload like this for the org list endpoint:

```json
{
  activeOrgId: 1234,
  defaultOrgId: 5678,
  orgs: [...]
}
```

but we instead chose to put the `isActive` and `isDefault` fields into the org structs instead, as we think it's easier to consume on the UI side.

The UI doesn't likely need those fields when querying a single org.

Similarly, the cluster information (provider/region*) takes extra work to compute, so we probably only want to include that in the single-org endpoint and not in the list.

Can we just do GraphQL now?

Reply:
I wish... In that case we will go ahead and split out the schemas.